homeassistant/components/bluetooth/__init__.py

```diff
@@ -314,7 +312,7 @@ async def async_update_device(
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up a config entry for a bluetooth scanner."""
-    manager: HomeAssistantBluetoothManager = hass.data[DATA_MANAGER]
+    manager = _get_manager(hass)
```

Comment:
Protected because custom integration keep importing this and than complaining when it breaks